test/system/700-play.bats

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bats   -*- bats -*-
+#
+# Test podman play
+#
+
+load helpers
+
+@test "podman play with stdin" {
+    run_podman play kube - <<< test.yaml
```

Comment:
This can't possibly work. I have to assume that you know this, that this is simply a placeholder. (Reason: `<<<` means "the next argument is a string variable"; what you probably want is `< $PODMAN_TMPDIR/test.yaml` but you need to create that first.)

Reply:
Fixed